docs/understanding-tracking-design/managing-data-structures-with-data-structures-builder/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "🆕 Managing data structures with the Data Structures Builder"
+date: "2023-03-01"
+sidebar_position: 85
+---
+
+:::info
+The Data Structures Builder is only available for BDP Cloud Customers
+:::
+
+To create a new Data Structure, first navigate to **Data Structures** in the menu and click the **'Create a Data Structure'** button.
+
+![](images/data-structures-1.png)
+
+Populate the general information, such as *Name,* and a *Description* and *Vendor*. *Vendor* allows you to organise your data structures, for example, by teams. Snowplow will automatically generate the Tracking Url to be referenced in your tracking code.
```

Comment:
This does not seem to offer much guidance beyond what the UI already expects the user to do. Can we give some examples here, using a realistic data structure someone might want to add?

Reply:
We're going to add a video tutorial soon where we go through an actual example.